### PR TITLE
Update custom augmentations tutorial

### PR DESCRIPTION
**Pull request by philippmwirth**
Update the custom augmentations tutorial to use transforms instead of the collate function.

### CODE REVIEW

File: docs/source/tutorials_source/package/tutorial_custom_augmentations.py [hunk 1]
```diff
@@ -168,6 +169,8 @@ def __call__(self, sample: torch.Tensor) -> torch.Tensor:
     ]
 )
 
+# Create a multiview transform that returns two different augmentations of each image.
+transform = MultiViewTransform(transforms=[view_transform, view_transform])
```
Comment (philippmwirth) [inline docs/source/tutorials_source/package/tutorial_custom_augmentations.py:173]: This is new.

File: docs/source/tutorials_source/package/tutorial_custom_augmentations.py [hunk 2]
```diff
@@ -222,17 +219,15 @@ def tiff_loader(f):
         return np.array(image)
 
 
-# create the dataset and overwrite the image loader
-dataset_train = LightlyDataset(input_dir=path_to_data)
+# Create the dataset with the custom transform and overwrite the image loader.
+dataset_train = LightlyDataset(input_dir=path_to_data, transform=transform)
```
Comment (philippmwirth) [inline docs/source/tutorials_source/package/tutorial_custom_augmentations.py:223]: Here we pass the transform now and a few lines below we no longer pass the collate function.

**Review by guarin — APPROVED** [review]
LGTM!

**Review by guarin — APPROVED** [review]
LGTM!

**Pull request merged**